install/ansible/playbooks/deploy_cluster.yml

```diff
@@ -260,3 +261,28 @@
     enabled: yes
     daemon-reload: yes
     state: restarted
+
+- name: set up htpasswd
+  become: true
+  block:
+    - name: change identity provider kind
+      lineinfile:
+        regexp: "      kind: AllowAllPasswordIdentityProvider"
+        line:   "      kind: HTPasswdPasswordIdentityProvider"
+        dest: "/var/lib/origin/openshift.local.config/master/master-config.yaml"
+    - name: set htpasswd file
+      lineinfile:
+        insertafter: "      kind: HTPasswdPasswordIdentityProvider"
+        line:        "      file: users.htpasswd"
+        dest: "/var/lib/origin/openshift.local.config/master/master-config.yaml"
```

Comment:
Hard coded these file paths since the location of openshift.local.config is not configurable in the provisioner.